lib/Frontend/CompilerInvocation.cpp

```diff
@@ -705,9 +712,13 @@ void CompilerInvocation::buildDWARFDebugFlags(std::string &Output,
                                               const ArrayRef<const char*> &Args,
                                               StringRef SDKPath,
                                               StringRef ResourceDir) {
+
+  llvm::SmallString<128> TDir;
+  llvm::sys::path::system_temp_directory(true, TDir);
```

Comment:
Can you put a comment here that says this isn't guaranteed to be the same temp directory as what the driver uses, but it's highly likely?

Reply:
sure